recipes/htmlmin/meta.yaml

```diff
@@ -0,0 +1,36 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.1.10" %}
+
+package:
+    name: htmlmin
+    version: {{ version }}
+
+source:
+    fn: htmlmin-{{ version }}.tar.gz
+    url: https://files.pythonhosted.org/packages/60/de/f42baab5b8cd8834311dfccf90bd7275bd8bab0f82780e244a5b05a165c1/htmlmin-0.1.10.tar.gz
```

Comment:
@daf can you change that ugly URL to `https://pypi.io/packages/source/h/htmlmin/htmlmin-{{ version }}.tar.gz`? (We are looking into changing `conda-skeleton` to avoid that.)